src/base/memory_monitor_sr.hpp

```diff
@@ -24,8 +24,15 @@
 #ifndef _MEMORY_MONITOR_SR_HPP_
 #define _MEMORY_MONITOR_SR_HPP_
 
+#include <stdint.h>
 #include <string>
 
+// IMPORTANT!!
+// This meta size is related with allocation byte align
+// Don't adjust it freely
+// 8 byte size + 4 byte tag + 4 byte magicnumber
+#define MMON_ALLOC_META_SIZE 16
```

Comment:
If this is used only for memory_monitor, 
then how about declare this constant in the class ? 
`static constexpr int alloc_meta_size = 16; `?

Reply:
It will be used allocation wrapping functions, like `cub_alloc()`.

Reply:
It seems you are probably concerned about the overhead of function calls..

Reply:
What I thought was, to easily understand that this constant is associated with the memory monitoring module, it would be better to declare it statically within the class. It facilitates namespace management as well.

Reply:
Oh I understand it. I change it to `static constexpr cubmem::MMON_ALLOC_META_SIZE`.